mantle/system/exec/exec.go

```diff
@@ -93,6 +96,14 @@ func (cmd *ExecCmd) Kill() error {
 	return err
 }
 
+func (cmd *ExecCmd) Signaled() bool {
+	if cmd.ProcessState == nil {
```

Comment:
I suspect the Go race detector will flag this; I believe the separate goroutine that is blocking in `Wait()` will be the one that writes to this data.  Probably hard to fix though without reworking things completely.  

If we were to rework things...I think we want a single stream of events from the qemu process effectively, then we could centralize the `sleep(1)` thing there.